components/list/list-item-generic-layout.js

```diff
@@ -225,20 +237,33 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 		}
 	}
 
+	_focusFirstCell() {
+		let cell = null;
+		let focusable = null;
+		let num = 1; // keep updated with highest cell-num
+		do {
+			cell = this.shadowRoot && this.shadowRoot.querySelector(`[data-cell-num="${num--}"]`);
+			if (cell) {
+				focusable = getFirstFocusableDescendant(cell) || focusable;
+			}
+		} while (cell);
+		focusable.focus();
+	}
+
 	_focusFirstRow() {
 		const list = findComposedAncestor(this, (node) => node.tagName === 'D2L-LIST');
 		const row = list.firstElementChild.shadowRoot.querySelector('[role="gridrow"]');
 		if (this.dir === 'rtl') {
-			row._focusLastItem();
+			row._focusLastCell();
 		} else {
-			row._focusNextCell(1);
+			row._focusFirstCell();
 		}
 	}
 
-	_focusLastItem() {
```

Comment:
I don't feel strongly about it, but I _think_ the "Item" term was being used because it is focusing the last item in the cell.

Reply:
That makes sense. Yeah it was certainly inconsistent. I can go back through and try to add that context everywhere if it makes sense.